src/handlers/event-bucketing/s3connect.ts

```diff
@@ -0,0 +1,54 @@
+import {
```

Comment:
There is a shared/utils/s3/ package - would it be appropriate to make some of this file a shared util?

Reply:
I don't think this would add value to the utils. It's basically an interface between the utils and this lambda with some added logic specific for this lambda. Was agreed with Mark during paired programming that it's better practice to try and split this interface logic out from the handler itself.